doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -103,6 +103,29 @@ If increasing the trace heap size does not help, either optimize the medium spee
 
 .. _partition_mgr_integration:
```

Comment:
This label should come after the following section and before the section : 'Partition manager integration'